core/blockchain_snapshot_test.go

```diff
@@ -388,15 +389,17 @@ func (snaptest *wipeCrashSnapshotTest) test(t *testing.T) {
 		SnapshotLimit:  256,
 		SnapshotWait:   false, // Don't wait rebuild
 	}
-	_, err = NewBlockChain(snaptest.db, config, snaptest.gspec, nil, snaptest.engine, vm.Config{}, nil, nil)
+	tmp, err := NewBlockChain(snaptest.db, config, snaptest.gspec, nil, snaptest.engine, vm.Config{}, nil, nil)
 	if err != nil {
 		t.Fatalf("Failed to recreate chain: %v", err)
 	}
+	tmp.Stop()
```

Comment:
The original intention here is to simulate an unclean shutdown without proper stop.

Reply:
Ah, ok, then I should change it to `stop` (only kills the threads, doesn't clean up)